### PR TITLE
Add CI about creation discussion w/o labels.

### PR DESCRIPTION
**Pull request by gnongs**
### 1. Type of change

Please delete options that are not relevant.

- [x] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [ ] others (anything other than above)

---

**Related:** #417 

**Summary**

This is CI that creates Discussion every two weeks.
---

Adding labels through CI will be uploaded soon.

### CODE REVIEW

**Comment by gnongs** [comment]
```sh
 ~/w/d/vatz make coverage                                                                                                                                     ci-create-discussion
go: downloading golang.org/x/net v0.7.0
go: downloading github.com/shirou/gopsutil v3.21.11+incompatible
go: downloading github.com/tklauser/go-sysconf v0.3.11
?   	github.com/dsrvlabs/vatz	[no test files]
ok  	github.com/dsrvlabs/vatz/cmd	6.975s	coverage: 18.0% of statements
ok  	github.com/dsrvlabs/vatz/manager/api	0.381s	coverage: 0.0% of statements [no tests to run]
ok  	github.com/dsrvlabs/vatz/manager/config	0.343s	coverage: 91.2% of statements
ok  	github.com/dsrvlabs/vatz/manager/dispatcher	0.356s	coverage: 7.8% of statements
ok  	github.com/dsrvlabs/vatz/manager/executor	0.694s	coverage: 68.9% of statements
ok  	github.com/dsrvlabs/vatz/manager/healthcheck	1.170s	coverage: 46.5% of statements
ok  	github.com/dsrvlabs/vatz/manager/plugin	9.044s	coverage: 51.0% of statements
?   	github.com/dsrvlabs/vatz/manager/types	[no test files]
?   	github.com/dsrvlabs/vatz/mocks	[no test files]
ok  	github.com/dsrvlabs/vatz/monitoring/prometheus	0.473s	coverage: 0.0% of statements
ok  	github.com/dsrvlabs/vatz/rpc	2.059s	coverage: 67.2% of statements
ok  	github.com/dsrvlabs/vatz/utils	0.929s	coverage: 7.1% of statements
```

File: .github/workflows/create-vatz-bi-weekly.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Create a next vatz discussion & milestone
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: "0 13 * * 3" # Every Wednesday at 1 p.m.
+env:
+  URL: https://api.github.com/repos/dsrvlabs/vatz/milestones # NEED TO CHANGE
+jobs:
+  create-bi-weekly:
```
Comment (xellos00) [inline .github/workflows/create-vatz-bi-weekly.yml:9]: Please, add some comment for each step that's going to help a lot for everyone to understand.

**Review by xellos00 — APPROVED** [review]

**Comment by xellos00** [comment]
@dsrvlabs/validator 
Please, review first, because it needs to be tested on DSRV environments.

File: .github/workflows/create-vatz-bi-weekly.yml (hunk 1 above)
Comment (gnongs) [reply .github/workflows/create-vatz-bi-weekly.yml:9]: > commit: 973154dfe649d307c5bfdb56995bd1662fd5e781

Add comment!

**Review by rootwarp — APPROVED** [review]
LGTM

**Pull request merged**